yajsapi/executor/index.ts

```diff
@@ -171,18 +175,22 @@ export class Executor {
 
   private emit;
 
+  private _network?: Network;
+  private _network_address?: string;
+
   /**
    * Create new executor
-   * 
+   *
    * @param task_package    a package common for all tasks; vm.repo() function may be used to return package from a repository
    * @param max_workers     maximum number of workers doing the computation
    * @param timeout         timeout for the whole computation
    * @param budget          maximum budget for payments
    * @param strategy        market strategy used to select providers from the market (e.g. LeastExpensiveLinearPayuMS or DummyMS)
    * @param subnet_tag      use only providers in the subnet with the subnet_tag name (env variable equivalent: YAGNA_SUBNET)
-   * @param driver          name of the payment driver to use or null to use the default driver; only payment platforms with the specified driver will be used (env variable equivalent: YAGNA_PAYMENT_DRIVER)
-   * @param network         name of the network to use or null to use the default network; only payment platforms with the specified network will be used (env variable equivalent: YAGNA_PAYMENT_NETWORK)
+   * @param payment_driver  name of the payment driver to use or null to use the default driver; only payment platforms with the specified driver will be used (env variable equivalent: YAGNA_PAYMENT_DRIVER)
```

Comment:
I believe we should still leave the old arguments in place and add a deprecation warning when they're used ... 

and only remove the old arguments in a subsequent minor release

Reply:
Yes, you are absolutely right. Too quickly I refactored the name to a more meaningful, forgetting about backwards compatibility.